### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Krzysiudan**
https://loudius.atlassian.net/jira/software/projects/LD/boards/3?selectedIssue=LD-51

Let's start together to work on README for Loudius. I've made a sketch for the beginning, but it would be awesome to get ideas from everyone. So feel free to add something from yourself, even making your own commit to this branch.
Don't take anything which is in the sketch as something not removable.

### CODE REVIEW

**Comment by Krzysiudan** [comment]
@kezc

**Comment by Krzysiudan** [comment]
@nowakweronika

File: README.md [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+architectures, solutions, and libraries. Functionalities of the app are simple. It uses GithubAPI
+and let's user ping his collaborators for a faster code review.
+
+## Contributing
+
+We believe that there is no ideal code and that every code can be improved. Therefore, we welcome
+every issue and new idea. We encourage you to open a new issue or pull request, as we can all learn
+from each other.
+
+## Experiments
```
Comment (kezc) [inline README.md]: When I was thinking about the readme I've came up writing about some features of the app, and that there're not many of them, so this makes it easy to understand the app :D Maybe we can reuse something

## **About the App**

Our app is a sample Android application designed to showcase some solutions to the architecture, networking layer, and Jetpack Compose. It provides a basic user interface and functionality that can be expanded upon to suit specific needs.

The app is open-source and intended for use by developers who are interested in expanding their knowledge about Android development. The main functionalities are:

- Login through GitHub OAuth
- List user’s pull requests
- Notifying reviewers to make a review

The small range of functionalities leads to the ease of understanding the code and conducting experiments with different development libraries and tools.

Comment (Krzysiudan) [reply README.md]: Sounds good to me. What do you think should we merge your description with: 


> This project serves as an example Android project and a playground for experimenting with new
architectures, solutions, and libraries. Functionalities of the app are simple. It uses GithubAPI
and let's user ping his collaborators for a faster code review.

or add it separately below? 🤔

**Comment by Krzysiudan** [comment]
@nowakweronika

File: README.md (hunk 1 above)
Comment (kezc) [reply README.md]: I think we can merge it :D Let's try to keep not too long

**Comment by Krzysiudan** [comment]
@nowakweronika

**Comment by Krzysiudan** [comment]
@kezc

File: README.md [hunk 2]
```diff
@@ -1,7 +1,99 @@
...
+Here's an example of an experiment that meets our rules:
+
+Branch Name: experiment/navigation-by-voyager-library
+Purpose: Check compose navigation with voyager library. Compare that with standard way.
+Goal: To resolve which navigation is better for compose. What are the pros and cons of each way.
+Method: Implement navigation with voyager library.
+
+## 🚀 Project setup
+
+In order to properly start the application and use it, the CLIENT_SECRET environment variable must be set on your computer.
```
Comment (Krzysiudan) [inline README.md]: **Thought: ** Maybe we should add some description of what is CLIENT_SECRET. That it is a GitHub API secret and in short where to get that.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,99 @@
...
+- Kotlin Coroutines
+- Retrofit
+- OkHttp3
+- Gson
+
+## 🧑🏻‍🎓 Contributing
+
+We believe that there is no ideal code and that every code can be improved. Therefore, we welcome
+every issue and new idea. We encourage you to open a new issue or pull request, as we can all learn
+from each other.
```
Comment (Krzysiudan) [inline README.md]: **Thought:** Maybe we should move contributing section to the end just before the license?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,99 @@
...
+related to Android development - UI, performance, architecture, libraries, and more.
+
+To create your own experiment, simply download our repository, create a new branch, and start
+exploring. Once you're done, create a pull request, and our community will review it. We believe
+that by sharing our ideas with each other, we can all learn and improve.
+
+We welcome all levels of experience in our community, whether you're a beginner or an expert. We're
+all here to learn and grow together.
+
+So come, join us in Loudius - Android experimental playground, and let's experiment together!
```
Comment (Krzysiudan) [inline README.md:46]: **Praise:** That sounds good for me 👌

File: README.md [hunk 5]
```diff
@@ -1,7 +1,99 @@
...
+knowledge about Android development. The main functionalities are:
+
+- login through GitHub OAuth,
+- list user’s pull requests,
+- ping collaborators for a faster code review.
+
+The small range of functionalities leads to the ease of understanding the code and conducting
+experiments with different development libraries and tools.
+
+<https://user-images.githubusercontent.com/72873966/228913610-6cce166f-37dd-4443-a8c3-83f6bf5fc489.mov>
```
Comment (Krzysiudan) [inline README.md]: **Thought** On the video there is no screen about app authorization in GitHub.  I'm not sure if we want that or not. On that screen, is there some sensitive information that you would like to hide?

File: README.md (hunk 4 above)
Comment (Krzysiudan) [inline README.md:46]: **Thought:** I'm wondering if it's not too long 🤔

File: README.md [hunk 6]
```diff
@@ -1,7 +1,99 @@
...
+- The experiment should not interfere with the stability of the existing codebase.
+
+### Example Experiment
+
+Here's an example of an experiment that meets our rules:
+
+Branch Name: experiment/navigation-by-voyager-library
+Purpose: Check compose navigation with voyager library. Compare that with standard way.
+Goal: To resolve which navigation is better for compose. What are the pros and cons of each way.
+Method: Implement navigation with voyager library.
```
Comment (Krzysiudan) [inline README.md]: **Question: ** Do you like that concept? Do you feel that's a good rule for experimenting?

File: README.md (hunk 5 above)
Comment (Krzysiudan) [inline README.md]: **Praise:** Having a video here it's a good idea 👌

Comment (nowakweronika) [reply README.md]: Yes, my password😂

File: README.md (hunk 4 above)
Comment (nowakweronika) [reply README.md:46]: I don't know, I think it's ok :) @kezc What do you think?

File: README.md (hunk 6 above)
Comment (nowakweronika) [reply README.md]: Yes, I think that's a good idea. Possible contributor will have to name what, why and how they want to implement something. I also think it's important to know it for us.

File: README.md (hunk 5 above)
Comment (nowakweronika) [reply README.md]: I changed the video😼

**Review by kezc — APPROVED** [review]

**Pull request merged**